resume.tex

```diff
@@ -24,26 +24,23 @@
 %==============================================================================
 
 \resumeSection{Work Experience}{
-  \experience{Software Development Engineer II}{MetLife Inc.}{2017-Present}{
-    \item Led a small team in building a proof-of-concept for a ``next-phase`` application built on Spring Boot and Angular 4, which was in turn used as a prototype for new projects company-wide
-	\item Consulted for a team building an application using my prototype as a template for a MongoDB-based plan repository with a Material Design UI
-	\item Implemented a report automation system through which users customize scheduled reports for the underwriting team
-    \item Enforced best practices and design principles by conducting code reviews
-  }
-  
+  \experienceHeader{MetLife Tech University Professional Program}{MetLife Inc.}{2014}\\
+  \experienceHeader{Associate Software Development Engineer}{MetLife Inc.}{2015-2016}\\
   \experienceHeader{Software Development Engineer I}{MetLife Inc.}{2016-2017}\\
-  \experience{Associate Software Development Engineer}{MetLife Inc.}{2015-2016}{
+  \experience{Software Development Engineer II}{MetLife Inc.}{2017-Present}{
+    \item Achieved the only perfect score for Technical Aptitude on my Tech University Professional Program team
+
+    \item Re-implemented part of one of MetLife's largest enterprise applications used by over 1300 claims administrators to handle over 800,000 disability claims a year, working on a small, Agile team
+    \item Reduced end-to-end search time by 42\% by replacing the outdated user interface with a Single Page Application built on Angular, Spring MVC, and a RESTful back end
+
     \item Developed an enterprise application used by underwriters to renew group benefits products for over 10,000 companies a month, each of which with up to 2000 employees
-    \item Pioneered MetLife's use of new technologies such as AngularJS and the Play Framework on an Agile team 
-	\item Implemented new features from requirements gathering to deployment by working closely with end-users, product owners, business analysts, and project managers
-    \item Taught Angular techniques to senior software engineers new to the framework as the team's designated subject matter expert for the client code
+    \item Pioneered MetLife's use of new technologies such as AngularJS and the Play Framework on an Agile team, taught Angular techniques to senior software engineers new to the framework, and was the "go-to" subject matter expert
     \item Led automated end-to-end testing with Selenium to reduce dependency on offshore QA and more closely adhere to Agile principles
+
+    \item Led a small team in building a proof-of-concept for a ``next-phase'' application built on Spring Boot and Angular 4, which was in turn used as a prototype for new projects company-wide, and consulted for other teams using it as a template
```

Comment:
Same here (diff sucks because I also moved it down to the bottom, to keep the accomplishments & job title stuff in matching orders).